### DISCUSSION TITLE
Add customization to styles

### DISCUSSION THREAD

**Discussion by Kiiv** [likec4/likec4#1246] [Ideas]
Hi,

don't know if I can create an issue directly for a new feature so I start here :)

It should be great to add some degree of customization to the styles :
- background fill or not
- text color
- border width

It sometime less aggressive to have just the stroke and the text  like the last example on this page when you have a lot of elements : https://docs.structurizr.com/dsl/cookbook/amazon-web-services/

**Comment by davydkov** [comment]
Hey @Kiiv 

> don't know if I can create an issue directly 

Sure you can, don't hesitate!
For this one - I created #1257
